src/components/Footer.tsx

```diff
@@ -1,82 +1,161 @@
-import { FaDev, FaDiscord, FaGithub, FaTwitter, FaYoutube } from "react-icons/fa";
+import { FaDev, FaDiscord } from "react-icons/fa";
+import OpenSaucedLogo from "../assets/openSauced-icon.png";
+
+// icons
+import
+{
+  AiOutlineTwitter,
+  AiOutlineGithub,
+  AiFillInstagram,
+  AiFillYoutube,
+} from "react-icons/ai";
+
+
+const footerContext = [
+  {
+    privacy: { url: "https://app.termly.io/document/privacy-policy/5e303854-d262-468a-80ec-54b645d01c2e", text: "Privacy" },
+    terms: { url: "https://app.termly.io/document/terms-of-use-for-saas/03e4e1c1-53ad-4fc4-b415-5c3f0e8c25ef", text: "Terms" },
+    status: { url: "https://api.opensauced.pizza/docs#/Health%20check%20service/healthStatusWeb", text: "Status" },
+  },
+  {
+    hot: { url: "https://hot.opensauced.pizza", text: "hot.opensauced.pizza" },
+    openSauced: { url: "https://opensauced.pizza", text: "opensauced.pizza" },
+  },
+  {
+    socials: [
+      { url: "https://twitter.com/saucedopen", icon: <AiOutlineTwitter className="text-2xl hover:text-light-slate-10 text-light-slate-9" /> },
+      { url: "https://github.com/open-sauced", icon: <AiOutlineGithub className="text-2xl hover:text-light-slate-10  text-light-slate-9" /> },
+      { url: "https://www.instagram.com/opensauced/", icon: <AiFillInstagram className="text-2xl hover:text-light-slate-10  text-light-slate-9" /> },
+      { url: "https://www.youtube.com/opensauced", icon: <AiFillYoutube className="text-2xl hover:text-light-slate-10  text-light-slate-9" /> },
+      { url: "https://discord.com/invite/U2peSNf23P", icon: <FaDiscord className="text-2xl hover:text-light-slate-10  text-light-slate-9" /> },
+      { url: "https://dev.to/opensauced/", icon: <FaDev className="text-2xl hover:text-light-slate-10  text-light-slate-9" /> },
+    ],
+  },
+];
 
 const Footer = (): JSX.Element => (
   <div>
-    <div className="bg-darkestGrey py-14 md:py-16">
-      <nav className="container">
-        <ul className="flex sm:flex-row text-xl font-righteous text-accent font-bold justify-center items-center cursor-pointer">
-          <li className="p-2 sm:mr-5">
+    <footer className="px-6 md:px-16 h-24 w-full bg-light-slate-2 transition">
+      <div className=" font-medium lg:border-t lg:py-8 lg:items-center lg:justify-between lg:gap-x-4 flex flex-col gap-y-4 lg:flex-row py-2 w-full">
+        <div className="text-center lg:text-left justify-center gap-1 flex items-center">
+          <div className="w-6 h-6 relative !min-w-[24px] min-h-[24px]">
+            <img
```

Comment:
any chance we can make this clickable? also happy if we open a new issue about it

Reply:
@0-vortex  Always there is a chance 😁